wcfsetup/install/files/lib/system/template/TemplateEngine.class.php

```diff
@@ -1045,14 +1045,18 @@ protected function getCompileFilePrefix(string $templateName): string
     private function getSharedTemplateGroupID(): int
     {
         if (!isset($this->sharedTemplateGroupID)) {
-            $sql = "SELECT  templateGroupID
-                    FROM    wcf1_template_group
-                    WHERE   templateGroupFolderName = ?";
-            $statement = WCF::getDB()->prepare($sql);
-            $statement->execute(['_wcf_shared/']);
+            $templateGroup = \array_find(
+                TemplateGroupCacheBuilder::getInstance()->getData(),
```

Comment:
Is there any reason why you don’t just use `$this->templateGroupCache` here? I even _explicitly_ pointed that out in #6316.